.github/workflows/set-date.yml

```diff
@@ -0,0 +1,85 @@
+name: Set the End Date in the project
+
+on:
+  issues:
+    types: [closed]
+jobs:
+  set_date:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Get project data
```

Comment:
In the `Get project data` step, the pipeline extracts the data from the project in a JSON file. Later we extract the `Project ID` and the `Date Field ID` of a field named "End Date" and set them to the env variables named `PROJECT_ID` and `DATE_FIELD_ID`